Bring consistency to sample titles (#192)
Esri/arcgis-maps-sdk-unity-samples: sample_project/Assets/SampleViewer/Samples/Geocoding/readme.md

> 5. To find a place, put a place name or address in the search bar and press Enter. For information about the type of locations that are supported by the Geocoding service, see the [ArcGIS REST APIs documentation](https://developers.arcgis.com/rest/geocode/api-reference/geocoding-find-address-candidates.htm).

Reviewer: ```suggestion
5. Use the search bar to find a place by name or address. For information about the type of locations supported by the Geocoding service, see the [ArcGIS REST APIs documentation](https://developers.arcgis.com/rest/geocode/api-reference/geocoding-find-address-candidates.htm).
```
Again, there's nothing wrong with what you have previously, but it's safe to assume all users know how to use a search bar.

> 6. To find an address of a place, drop the marker to your point of interest by clicking in the viewport while holding Shift, and you will see the address displayed above the marker.

Reviewer: ```suggestion
6. To find a place's address, drop the marker on your point of interest by clicking in the viewport while holding Shift, the address will display above the marker.
```
